
syntax = "proto3";
package pkcamera;

service PkCamera {
   rpc GetCamStatus(Field) returns (CamStatus) {}
}

message Field {
    string name = 1;
}

message CamStatus {
    int32 status = 1;
    string msg = 2;
}
